src/api/config.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import { Config } from '../entities/config';
+import logger from '../logger';
+
+export const config = express.Router();
+
+config.get('/', async (req, res) => {
+  const { configKey } = req.query;
+
+  if (!configKey || !configKey.trim()) {
```

Comment:
Can we support having no `configKey` and get all of them back? That way I can show them all on the page

Reply:
Yep, that makes sense